### DISCUSSION TITLE
DAO Creative Universe @ ETH Denver 👋

### DISCUSSION THREAD

**Discussion by thelastjosh** [metagov/daostar#41] [General]
# Welcome to the DAO Creative Universe

You're probably having your photo taken now and having it added to the DAO Creative Universe map. Here are some other things you can do to contribute and participate in [DAOstar](https://daostar.org) and the [DAO Creative Universe](https://github.com/metagov/daostar/tree/main/creative-universe).

## What can I do here?

1. Introduce yourself (see below)
2. [Submit an idea](https://github.com/metagov/daostar/discussions/categories/ideas) for a DAO Improvement Proposal (DAOIP)
3. Change the [DAO Creative Universe](https://github.com/metagov/daostar/tree/main/creative-universe) itself.

## Introduce Yourself

We want to create a space for people working in DAOs to meet each other. Leave a comment below and describe you interest in DAOs and how you think they could be improved.

**Note**: We'll be looking through these introductions. If you want someone to get in touch with you, _please be specific about your interest in DAOs_. See some examples in the comment thread below!

**Comment by thelastjosh** [comment]
Hi everyone,

I'm Josh, and I'm a computer scientist and a mathematician. To be clear, I'm not the useful kind of computer scientist that can actually build software. I mostly model computers and algorithms and AI using a branch of math called applied category theory, though I also do a fair amount of work modeling social and economic designs. I started and help run a nonprofit called [Metagov](https://metagov.org). I'm interested in working on DAOIPs focused on improving governance, especially the amendment procedures for modifying digital governance, both on-chain and off-chain.

Look forwarding to working with everyone!

Josh

**Comment by ntnsndr** [comment]
Hi, I'm Nathan—a professor of media studies at CU Boulder and a Metagov board member.

**Comment by DSB-117** [comment]
Hello, my name is Dakota & I am the founder of the InvArch Network, a layer-1 blockchain highly optimized for housing the DAO economy.

**Comment by arrudagates** [comment]
I'm Gabe, a ChaosDAO member + CHAD at GMorDie DAO & 2022 Golden Birdie winner developing InvArch Network on porkydot because porkydot gud

**Comment by davedavecoolname** [comment]
Hey there! I'm Davedave, a Business Development Specialist here at Eth Denver with the PARSIQ team. I was just trying to get these guys to use our API and got dragged in to a totally cool project!

**Comment by shanmukhipriya99** [comment]
Hey
This is Shanmukhi. I’m a recent graduate focusing on getting a jog in the WEB3 space.
I’m here at ETHDenver hoping to get a jump start on the ZKP concept and it has been quite fruitful so far.
I’d love to connect to people with similar interests. Take a look at my profile for my contact info.😇

**Comment by skuhlmann** [comment]
Sam K. Building at DAOhaus, Raid Guild, MetaCartel.

**Comment by dekanbro** [comment]
Hi!

Dekan from DAOhaus  🟩 🔺🟡

https://daohaus.club/

**Comment by ppong** [comment]
Hi I am Peter, I am with nestwallet.xyz. We are an orange DAO portfolio company!

**Comment by unfriendlyamendment** [comment]
Hi! This is Hazel. I hang out with [DAOstar](https://daostar.org/), [Plurigrid](https://plurigrid.xyz/) and [KERNEL](https://www.kernel.community/) among a few other projects, prev. [SCRF](https://www.smartcontractresearch.org/). I have a political theory background, and I'm generally interested in DAOs, social choice theory, agency, and community self governance. 

I'm currently working on building out new ways of unionising with [theLaborDAO](https://twitter.com/thelabordao) and shipping a DAO contributor compensation strategy research.

**Comment by Daokey2022** [comment]
Hi we are DAOKEY a NFT art DAO project Dan we are looking for new ideas to invest in. https://linktr.ee/daokey

**Comment by amsos** [comment]
Hey! I'm Aaron from [Govrn](govrn.io) where we're building a contribution tracking protocol for DAO contributors.  In addition to that I've been helping work on the DAOStar attestation standard, as well as contributing to DAOs in the metacartel ecosystem.

**Comment by davedavecoolname** [comment]
Hey guys, David from PARSIQ here. If you guys need an API for blockchain
data for no charge to help you with your projects, reach out to me and I'll
get you set up.

Good luck!!

**Comment by RolfAris** [comment]
Rolf here from MCV, Cultur3, currently writing a book on DAOs leveraging my PhD in org theory

**Comment by Florian-ootopia** [comment]
Florian here from threefold.io ! Amazing work

**Comment by Alaindrea** [comment]
Hi 👋 I am Alaindrea repping Cityroots DAO as the operations founder. Happy to meet some other DAOs

**Comment by ThaDoubleB** [comment]
DoubleB checking in from the Bankless & DAOpunks community
